global_install/hooks/.pre-commit-config.yaml

```diff
@@ -10,4 +10,9 @@ repos:
     hooks:
     -   id: detect-secrets
         args: ['--baseline', '.secrets.baseline']
-        exclude: .*/tests/.*
+        exclude: .*/tests/.*|
+              (?x)(
+                     ^package-lock.json$/|
+                     ^Pipfile/|
+                     ^pyproject.toml
```

Comment:
Any more common hash-y files you can think of?

Reply:
I think this should be added to a step in the documentation so users explicitly know which files are, and are not scanned for committed secrets. While this helps adoption it adds a non obvious behaviour.

Reply:
Already there, if you remember, you razzed me for my `cat` use :laughing: 
https://github.com/alphagov/gds-pre-commit#ignoring-secrets

Reply:
I REMOVED THE :cat:

Reply:
Should these additional ignore be added to the documentation too then?

Reply:
Yeah, good point. It's actually in the default example config, which I have PR'd. https://github.com/Yelp/detect-secrets/pull/340